src/dvclive/live.py

```diff
@@ -370,6 +380,43 @@ def step(self, value: int) -> None:
         self._step = value
         logger.debug(f"Step: {self.step}")
 
+    def monitor_system(
+        self,
+        interval: float = 0.05,  # seconds
+        num_samples: int = 20,
+        directories_to_monitor: Optional[Dict[str, str]] = None,
```

Comment:
Why not default to `{"main": "/"}`?

Reply:
I think it is really bad practice to default a dict (or any mutable structure) :)